torch/csrc/jit/passes/quantization.cpp

```diff
@@ -745,17 +745,13 @@ bool isBiasOfConvOrLinear(Value* v) {
   return result;
 }
 
-bool isWeightOfConvOrLinear(Value* v) {
```

Comment:
we still need this, we have other code that relies on this function

Reply:
I've moved the logic inside `isWeight`. All the existing tests are passing with this change.